src/Subscriptions.tsx

```diff
@@ -19,6 +19,7 @@ export interface SubscriptionResult<TData = any> {
 export interface SubscriptionProps<TData = any, TVariables = OperationVariables> {
   query: DocumentNode;
   variables?: TVariables;
+  shouldResubscribe?: boolean;
```

Comment:
We need to add a test for this in the Subscription component test suite in a follow-up